vscode/test/suites/extensionUtils.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as vscode from "vscode";
+import { type ExtensionApi } from "../../src/extension";
+
+// Q# extension log level. Increase this for debugging.
+const extensionLogLevel = "warn";
+
+export async function activateExtension() {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const ext = vscode.extensions.getExtension("quantum.qsharp-lang-vscode-dev")!;
```

Comment:
The extension id will be different between dev and release. Is that safe here? (And if so, please add a comment as to why)

Reply:
(I'm guessing it's because it's test code and thus we assume the 'dev' extension, but wouldn't hurt to be clear)

Reply:
(I wouldn't worry about it if it means pushing more commits and waiting though, so resolving for now)